distribution/docs/src/main/resources/_contents/_installing/controlling-file-system-access-contents.adoc

```diff
@@ -50,6 +50,8 @@ Set directory permissions to protect the ${branding} from unauthorized access.
 
 * Change ownership of `<${branding}_HOME>`
 ** `chown -R ${branding-lowercase}-user <${branding}_HOME>`
+* Create instances sub-directory if does not exist
+** `mkdir -p <${branding}_HOME>/instances`
```

Comment:
yeah I didn't want it to error if the instances directory already existed so their wouldn't be any confusion.